kaldb/src/main/java/com/slack/kaldb/chunkManager/IndexingChunkManager.java

```diff
@@ -114,7 +114,7 @@ public IndexingChunkManager(
       MeterRegistry registry,
       BlobFs blobFs,
       String s3Bucket,
-      ListeningExecutorService rollOverExecutorService,
+      ListeningExecutorService rolloverExecutorService,
```

Comment:
any reason why we change 
```
rollOverExecutorService to rolloverExecutorService,
```

but in the previous file we use camelCase?

```
chunkRolloverStrategy -> chunkRollOverStrategy
```

Reply:
The rule for errorprone is they just need to be consistent, so I fixed the locations they flagged. Personally I don't like splitting rollover into two words but that's for another day. :D